src/Covid19Radar.Api/Services/DeviceValidationAppleService.cs

```diff
@@ -102,8 +102,7 @@ public async Task<bool> Validation(IAppleDeviceVerification deviceVerification,
                 //}
                 if (response.StatusCode != System.Net.HttpStatusCode.OK)
                 {
-                    // FIXME: When call iOS Device check, return error sometimes, Until the cause is known, ignored device check
-                    return true;
+                    return false;
                 }
```

Comment:
ここはシンプルにそのままreturnしてしまっても良い気がします。
一方、この部分は前回のように失敗してもひとまず通すようにしたり、今後ログを仕込んだり、いろいろやることがありそうなので、あまり大きく変えずにいまの構造を保ちたいというのも理解はできるので、お好みでと言うことで…

```suggestion
                return (response.StatusCode != System.Net.HttpStatusCode.OK);
```

Reply:
せっかくご提案いただいたので、そのままreturnするようにします。

Reply:
削除行があると、ここからCommitできないようなので、私の方で同様の修正をしてpushしました。